pareto/strategic_water_management/run_strategic_model.py

```diff
@@ -70,15 +71,18 @@
     "CST",
     "CCT",
     "CKT",
+    "Elevation",
     "CompletionsPadOutsideSystem",
     "DesalinationTechnologies",
     "DesalinationSites",
     "TruckingTime",
     "CompletionsDemand",
     "PadRates",
     "FlowbackRates",
+    "WellPressure",
     "NodeCapacities",
     "InitialPipelineCapacity",
+    "InitialPipelineDiameters",
```

Comment:
Is the onus on the user to ensure that the data entered in "InitialPipelineDiameters" is compatible with the data for "InitialPipelineCapacity"?

Reply:
I believe so. can explain offline

Reply:
Following up on this comment - should the data in the "InitialPipelineDiameters" tab only be in one direction (like for "InitialPipelineCapacity"), and not two directions? This screenshot is from the "InitialPipelineDiameters" tab in "strategic_toy_case_study.xlsx": 
![image](https://github.com/project-pareto/project-pareto/assets/26798258/38db5174-94c6-4b9a-b613-227f6e73a930)

I would think that e.g. you want to remove the entry for N02 -> N01, leaving instead just the entry for N01 -> N02, but I'm not sure.

Reply:
@nareshsusarla I would still like an explanation here - perhaps best for us to discuss on a call though.

Reply:
We discussed this on a call and agreed that the structure of the "InitialPipelineDiameters" tab should be changed as I suggested above, to remove symmetrical entries (similar to the "InitialPipelineCapacity" tab).